saas/models.py

```diff
@@ -3955,7 +3948,7 @@ def is_broker(organization):
         return import_string(settings.IS_BROKER_CALLABLE)(organization_slug)
     return get_broker().slug == organization_slug
 
-
+# TODO seems like there is a duplicate in utils.py:full_name_natural_split
```

Comment:
I didn't remember about this `split_full_name` function. It is deprecated and should be replaced by `full_name_natural_split`. (this might be a reason for some weird issues we see sometimes when people invite contributors).